src/main/java/stirling/software/SPDF/model/api/misc/ExtractHeaderRequest.java

```diff
@@ -16,4 +18,9 @@ public class ExtractHeaderRequest extends PDFFile {
             required = false,
             defaultValue = "false")
     private boolean useFirstTextAsFallback;
+
+    private MultipartFile fileInput;
```

Comment:
this multiFile input is already part of this class... its from PDFFile that's extended